folly/docs/Futures.md

```diff
@@ -39,7 +39,7 @@ void foo(int x) {
 
 This would print:
 
-```
+```cpp
```

Comment:
This one should not be highlighted as C++.

Reply:
Sorry, It was just copy paste.